### ISSUE TITLE
Gradients not showing

### ISSUE THREAD

**Issue by dmnphpkmnr** [robb0wen/synthwave-vscode#258]
Looking at the screenshots of the theme, there should be at least two gradients:

1. In the tab bar, the active tab should be highlighted by a line that is filled with a gradient
2. In the sidebar, if there are unsaved changes or notifications, the counter bubble should be filled with a gradient

None of these are working. Both are filled with solid colors.

I am using VSCode on macOS Monterey. I should have the latest version of VSCode and macOS and I installed the theme from the VSCode marketstore inside the app.

**Comment by elijaholmos** [comment]
I am experiencing this issue as well on Windows 10, VSCode 1.68.0. No gradient anywhere

**Comment by mich4s74** [comment]
Yeah i just installed it because i wanted to see how the gradients were applied to the theme.. But they don't show up at all anywhere like the screenshots of the theme.. I don't know why at all
